# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/greetd/var-run.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- greetd-0.9.0/greetd/src/config/mod.rs.vanilla	2024-01-06 12:20:40.281396645 +0100
+++ greetd-0.9.0/greetd/src/config/mod.rs	2024-01-06 12:20:50.005159775 +0100
@@ -5,7 +5,7 @@
 
 use super::error::Error;
 
-const RUNFILE: &str = "/run/greetd.run";
+const RUNFILE: &str = "/var/run/greetd.run";
 
 #[derive(Debug, Eq, PartialEq)]
 pub enum VtSelection {
--- greetd-0.9.0/greetd/src/server.rs.vanilla	2024-01-06 12:20:55.838946860 +0100
+++ greetd-0.9.0/greetd/src/server.rs	2024-01-06 12:21:07.842380734 +0100
@@ -175,7 +175,7 @@
 
 impl Listener {
     fn create(uid: Uid, gid: Gid) -> Result<Listener, Error> {
-        let path = format!("/run/greetd-{}.sock", getpid().as_raw());
+        let path = format!("/var/run/greetd-{}.sock", getpid().as_raw());
         let _ = std::fs::remove_file(&path);
         let listener =
             UnixListener::bind(&path).map_err(|e| format!("unable to open listener: {}", e))?;
